### DISCUSSION TITLE
How to Change Mouse Cursor Hover Style?

### DISCUSSION THREAD

**Discussion by leonasdev** [wezterm/wezterm#3581] [Q&A] [answered]
os: Windows 11

Is there any setting to change the mouse cursor style when hovering over an element? Currently, the cursor style is set to "text" when hovering, but is there a way to change it to "default"?

current: 
![image](https://user-images.githubusercontent.com/39915562/233681445-60db6177-58ea-4ca2-8c71-e12754511e74.png)

expected:
![image](https://user-images.githubusercontent.com/39915562/233681499-42c11841-5852-4e7c-913c-1ddc36e7d209.png)

**Comment by wez** [comment, marked as answer]
There is no option for this

**Reply by leonasdev** [reply]
Sadge...
